lib/mdl/rules.rb

```diff
@@ -516,6 +516,7 @@ def check_blockquote(errors, elements)
     fence = nil
     prev_line = ""
     doc.lines.each_with_index do |line, linenum|
+      next if line.match(/{:toc}/)
```

Comment:
Thanks for the documentation. I'm fine with this change in theory, but this feels a bit overly broad. I much suggest `line == '{:toc}'` or perhaps if whitespace is allowed before it `line.strip == '{:toc}'`

Reply:
@jaymzh done.